js.js

```diff
@@ -0,0 +1,40 @@
+const books = [];
+const addBtn = document.getElementById('add-new-book');
+const booksoutput = document.querySelector('#booksoutput');
+
+function outputbooklist() {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.